### PR TITLE
No token ring anymore

### PR DESCRIPTION
**Pull request by mchf**
bsc#1167474

### CODE REVIEW

File: src/include/dhcp-server/commandline.rb [hunk 1]
```diff
@@ -124,7 +124,7 @@ def initialize_dhcp_server_commandline(include_target)
               "The hardware type of the host with a fixed address"
             ),
             "type"     => "enum",
-            "typespec" => ["ethernet", "token-ring"]
+            "typespec" => ["ethernet"]
```
Comment (jreidinger) [inline src/include/dhcp-server/commandline.rb:127]: does this make sense at all? maybe for backward compatibility?

File: src/include/dhcp-server/dialogs2.rb [hunk 2]
```diff
@@ -1733,11 +1718,9 @@ def HostManagementInit(key)
...
           Id(id2),
           id2,
-          Ops.get(opts, "ip", ""),
-          Ops.get(opts, "hardware", ""),
-          Ops.get(opts, "type", "ethernet") == "ethernet" ?
-            _("Ethernet") :
-            _("Token Ring")
+          opts.fetch("ip", ""),
+          opts.fetch("hardware", ""),
+          _("Ethernet")
```
Comment (jreidinger) [inline src/include/dhcp-server/dialogs2.rb:1723]: maybe we can remove this column in table?

File: src/include/dhcp-server/dialogs2.rb [hunk 3]
```diff
@@ -1805,19 +1777,18 @@ def CheckMacAddrFormat
       addr_type = Convert.to_string(
         UI.QueryWidget(Id("network_type"), :CurrentButton)
```
Comment (jreidinger) [inline src/include/dhcp-server/dialogs2.rb:1778]: does this widget still exists?

File: src/include/dhcp-server/commandline.rb (hunk 1 above)
Comment (mchf) [reply src/include/dhcp-server/commandline.rb:127]: I was not sure. I've kept it for compatibility reasons only. AFAIK it is not mandatory anyway.

File: src/include/dhcp-server/dialogs2.rb (hunk 2 above)
Comment (mchf) [reply src/include/dhcp-server/dialogs2.rb:1723]: Another place where I was not sure. I'll remove it.

File: src/include/dhcp-server/dialogs2.rb (hunk 3 above)
Comment (mchf) [reply src/include/dhcp-server/dialogs2.rb:1778]: hm, it escaped ... will fix

File: src/include/dhcp-server/commandline.rb (hunk 1 above)
Comment (jreidinger) [reply src/include/dhcp-server/commandline.rb:127]: so at least please add comment that it is just for backward compatibility and maybe after string freeze we can add to help text that it is now noop.

**Comment by mchf** [comment]
screenshots are in trello ... I'll add it even here.

**Pull request merged**